modules/coreos-layering-configuring.adoc

```diff
@@ -32,7 +32,8 @@ For example, the following Containerfile creates a custom layered image from an
 # Using a 4.12.0 image
 FROM quay.io/openshift-release/ocp-release@sha256... <1>
 #Install hotfix rpm
-RUN rpm-ostree override replace https://example.com/hotfixes/haproxy-1.0.16-5.el8.src.rpm && \ <2>
+RUN rpm-ostree cliwrap install-to-root / \
```

Comment:
The problem is that this command is currently only necessary when overriding the kernel.  It has other side effects (it won't break things really), but still, I'd hoped to constrain it to that use case.

So I'd prefer instead that we copy the existing example for kernel overrides into the doc specifically.

Reply:
gotcha, i've updated this with the kernel override example, and removed the that command from the other location that shows a Containerfile for layering with a Hotfix